src/Ocis.php

```diff
@@ -722,6 +737,56 @@ public function getUserById(string $userId): User
         return new User($apiUser);
     }
 
+    /**
+     * Create a new education user (if the user has the permission to do so)
+     *
+     * @return EducationUser
+     */
+    public function createEducationUser(
+        string $displayName,
+        string $onPremisesSAMAccountName,
+        ?string $surname = null,
+        ?string $givenName = null,
+        ?string $mail = null,
+        ?string $primaryRole = null,
+        ?string $issuer = null,
+        ?string $issuerAssignedId = null,
+    ): EducationUser {
+        if(isset($this->educationGuzzle)) {
+            throw new \InvalidArgumentException(
+                "Guzzle client for education user is not initilized."
```

Comment:
that is generally true, but the real reason why the client is not initialized is that there was no access token provided.
```suggestion
                "This function cannot be used because no authentication token was provided for the educationUser endpoints."
```